earthly/rust/scripts/std_build.sh

```diff
@@ -42,6 +44,11 @@ status $rc "Checking Documentation tests all pass" \
 status $rc "Checking Benchmarks all run to completion" \
     cargo bench --all-targets; rc=$?
 
+## Generate dependency graphs
+cargo depgraph --workspace-only --dedup-transitive-deps > target/doc/workspace.dot
+cargo depgraph --dedup-transitive-deps > target/doc/full.dot
+cargo depgraph --all-deps --dedup-transitive-deps > target/doc/all.dot
+
 ## Generate Module Trees for documentation purposes.
 # cargo modules generate tree --orphans --types --traits --fns --tests --all-features --lib
```

Comment:
Would be good to enable these as well and add them to the docs, as they will show us the internal structure of our modules.
We can generate a graph, as well as the tree.  I think both are useful.
For the `--bin` target we will need to know the name of the binary.  We could pass it as a parameter to std-build.sh and get it in the Earthfile `BUILD` UDC.  We would then need to supply the binary name.  If no binary name was supplied we could skip generation for `--bin` and only generate for --lib.

Reply:
done